Gemfile.lock

```diff
@@ -6,27 +6,27 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.2.5)
-    json (1.8.3)
-    json (1.8.3-java)
-    rake (10.4.2)
-    rake-compiler (0.9.5)
+    diff-lcs (1.3)
+    json (1.8.6)
+    json (1.8.6-java)
+    rake (12.3.1)
```

Comment:
Actually, this is breaking the build for Ruby 1.8:

```
rake-12.3.1 requires ruby version >= 2.0.0, which is incompatible with the
current version, ruby 1.8.7p371
```

Though I'm theoretically fine with dropping the ability to run future tests on 1.8, is there some rake version that will continue to satisfy all Ruby versions?